src/tree_sitter/parser.h

```diff
@@ -123,6 +123,7 @@ struct TSLanguage {
     unsigned (*serialize)(void *, char *);
     void (*deserialize)(void *, const char *, unsigned);
   } external_scanner;
+  const TSStateId *primary_state_ids;
```

Comment:
Hmm, does this stem from an older tree-sitter cli generation? If so, it'd be nice to see if it shows up generating now (and if not, remove it :) )